src/content/docs/pt-br/basics/astro-pages.mdx

```diff
@@ -97,3 +100,75 @@ Para uma página customizada de erro 404, você pode criar um arquivo `404.astro
 
 Isso irá construir uma página `404.html`. A maioria dos [serviços de deploy](/pt-br/guides/deploy/) irão a encontrar e utilizar.
 
+## Parciais de páginas
+
+<p><Since v="3.4.0" /></p>
+
+:::caution
+É esperado que parciais de páginas sejam utilizadas em conjunto com uma biblioteca front-end, como [htmx](https://htmx.org/) ou [Unpoly](https://unpoly.com/). Você também pode utiliza-las caso se sinta confortável escrevendo JavaScript front-end low-level. Por conta disso elas são consideradas um recurso avançado.
+
+Adicionalmente, parciais não devem ser usadas se o componente possuir estilos com escopo ou scripts, pois esses elementos serão removidos do HTML gerado. Se você precisa de estilos com escopo, melhor utilizar páginas padrões sem parciais de páginas em conjunto com uma biblioteca frontend que sabe mesclar conteúdo no head.
+:::
+
+Parciais são componentes de página localizados em `src/pages/` que não tem como objetivo serem renderizados em páginas completas.
+
+Assim como componentes localizados fora dessa pasta, esses arquivos não incluem a declaração `<!DOCTYPE html>` automaticamente, nem qualquer outro conteúdo do `<head>`, como estilos com escopo ou scripts.
+
+Porém, por estarem localizados no diretório especial `src/pages/`, o código HTML gerado fica disponível em uma URL que corresponde ao caminho do arquivo. Isso permite que uma biblioteca de renderização(ex: htmx, Stimulus, jQuery) possa acessa-lo no cliente e carregar sessões do HTML dinamicamente em uma página, sem a necessecidade de um recarregamento ou navegação entre páginas.
```

Comment:
```suggestion
Porém, por estarem localizados no diretório especial `src/pages/`, o código HTML gerado fica disponível em uma URL que corresponde ao caminho do arquivo. Isso permite que uma biblioteca de renderização (ex: htmx, Stimulus, jQuery) possa acessá-lo no cliente e carregar sessões do HTML dinamicamente em uma página, sem a necessecidade de um recarregamento ou navegação entre páginas.
```

Curiosity side note, `(e.g. ...)` is valid in Portuguese as well, it is a latin abbreviation meaning "exempli gratia", which translates to "for example" (or more formally to "by virtue of the example"). It can be used across all languages with a latin script just like `p.s.:` for "post scriptum" meaning written after the text.

Using `ex:` is an informal variation, the correct abreviation (and by that I mean what would be accepted on a formal academic text, not the case here) for Portuguese would be `p. ex.:` or `e.g.:`.

Reply:
Nice, good to know! Had no idea about the "e.g." part